ci/before_install.sh

```diff
@@ -23,20 +23,20 @@ echo 'python info:'
   2>&1 pyenv versions || true
 ) | sed 's/^/  /'
 
+if [[ "${TRAVIS_OS_NAME}" == osx ]]; then
+    echo "Install Python 2."
+    brew install python@2
```

Comment:
why bother with this? we weren't messing with python2 before.

Reply:
Otherwise we'll have no `pip2.7`.

Reply:
Ok, but with the current changes we are no longer installing python3. Was that intentional? `brew install python3` was removed, is python3 assumed to exist already?

Reply:
That's the thing. Travis always installed "python" via homebrew by default. That used to be Python 2 and is Python 3 now. So we now have to upgrade Python 3 and install Python 2 for `pip2.7`.